test/testWs/src/test_project_1/package.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>test_project_1</name>
+  <version>0.0.1</version>
+  <description>test_project_1</description>
+
+  <maintainer email="paul.furgale@gmail.com">Paul Furgale</maintainer>
```

Comment:
Hm, we should have canonic owner names (more roles than persons) that belong to the lab or its groups. Otherwise we have an extra maintenance load whenever somebody leaves overhead. In this particular example I could use a dummy maintainer (which could be stable for a long time), as it is not a real project? (only a test project).